src/components/Navbar/Navbar.tsx

```diff
@@ -1,52 +1,40 @@
-import styled from "styled-components"
-
-const NavbarContainer = styled.nav`
-width: 100%; 
-height: 172px; 
-background-image: url('/images/bg-sidebar-mobile.svg'); 
-background-size: cover; 
-display: flex; 
-justify-content: center; 
-align-items: center; 
-`
-
-const StepList = styled.ul`
-display: flex; 
-gap: 20px; 
-margin-bottom: 50px; 
-`
-
-const StepButton = styled.button`
-width: 33px; 
-height: 33px; 
-border-radius: 50%; 
-font-weight: 700; 
-font-size: 14px; 
-color: ${props => props.theme.colors.white}; 
-background-color: ${props => props.theme.colors.lightBlue};
-border:  ${props => `1px solid ${props.theme.colors.white }`} 
-
-`
+import { NavbarContainer, StepList, StepButton } from './NavbarStyles';
+import { useLocation } from 'react-router-dom';
 
 const Navbar: React.FC = () => {
-
-
-    const stepNumber = [1,2,3,4]
-
-    return (
-        <NavbarContainer>
-            <StepList>
-            {stepNumber.map(stepNumber => (
+  const stepNumber = [1, 2, 3, 4];
+  const location = useLocation();
+
+  let currentStep: number;
+
+  switch (location.pathname) {
```

Comment:
A very important thing when developing an application is having so called "single source of truth". That said, you should have one place in application that defines the relationship between steps and URLs, so you should have one object in the app, being e.g.:

```
const steps = [{
  id: 1,
  label: "Personal information",
  url: "/personalinformation"
}]
```
etc...

and all the other things in the app you achieve by modyfying this object via maps, reduces, filters, includes, etc...